spec/models/concerns/hyku_addons/user_behavior_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+RSpec.describe HykuAddons::UserBehavior, type: :model do
+  subject(:user) { create(:user) }
+
+  describe ".with_public_profile" do
+    subject(:query) { User.with_public_profile }
+    let(:public_user) { create(:user, display_profile: true) }
+
+    before do
+      user && public_user
+    end
+
+    it "doesn't return private users" do
+      expect(query).to include(public_user)
+      expect(query).not_to include(user)
+    end
+
+    it "doesn't affect normal queries" do
```

Comment:
I don't think this test is required.  It would only affect normal queries if `default_scope` was defined.  Is it defined somewhere?

Reply:
@thenapking honestly, i dont trust Active Fedora, so i wanted to be sure that this worked as I would expect.